UnityProject/Assets/Scripts/Chemistry/ReactionLib.cs

```diff
@@ -70,167 +70,156 @@ private static void JsonImportInitialization()
 
 	private static void ChemInitialization()
 	{
-		foreach(var reaction in ChemistryGlobals.reactions)
+		foreach (var reaction in ChemistryGlobals.reactions)
 		{
-			foreach (string chemical in reaction.ReagentsAndRatio.Keys)
+			foreach (string reagent in reaction.ReagentsAndRatio.Keys)
 			{
-				if (!ChemistryGlobals.reactionsStoreDictionary.ContainsKey(chemical))
+				if (!ChemistryGlobals.reactionsStoreDictionary.ContainsKey(reagent))
 				{
-					ChemistryGlobals.reactionsStoreDictionary[chemical] = new HashSet<Reaction>();
+					ChemistryGlobals.reactionsStoreDictionary[reagent] = new HashSet<Reaction>();
 				}
-				ChemistryGlobals.reactionsStoreDictionary[chemical].Add(reaction);
+				ChemistryGlobals.reactionsStoreDictionary[reagent].Add(reaction);
 			}
 		}
 	}
 }
 
 public static class Calculations
 {
+	private const int REAGENT_SIGNIFICANT_DECIMAL_POINTS = 3; // Decimal point all reagents will be rounded to
```

Comment:
this is not used anywhere?

I assume it's supposed to go with this "if (reagent.Value <= 0.001)"